lib/proto.js

```diff
@@ -71,10 +77,13 @@ module.exports = function () {
 }
 
 const resetChildingsFormatter = bindings => bindings
-function child (bindings) {
+// TODO: formatters, serializers, customLevels should move to options
+// and depercation warning should emit
```

Comment:
TODO?

Reply:
Yes, those options in bindings should move to options as described in the original issue. But it can be done in another PR. I will raise another PR once this get merged, it can also reduce the complexity in a single PR.